lib/kernel-manager.coffee

```diff
@@ -69,7 +69,7 @@ class KernelManager
 
         catch e
             unless e.code is 'ENOENT'
-                console.log 'KernelManager: Cannot start existing kernel:\n', e
+                console.error 'KernelManager: Cannot start existing kernel:\n', e
```

Comment:
Don't think this should be logged as an error, because `./hydrogen/connection.json` is optional.

Instead, I would reword the log message, e.g.
```coffeescript
console.error 'KernelManager: Didn't find a `./hydrogen/connection.json`:\n', e
```